src/overlays/actors/ovl_Obj_Moon_Stone/z_obj_moon_stone.h

```diff
@@ -5,9 +5,13 @@
 
 struct ObjMoonStone;
 
+typedef void (*ObjMoonStoneActionFunc)(struct ObjMoonStone*, GlobalContext*);
+
 typedef struct ObjMoonStone {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x54];
+    /* 0x144 */ char unk_144[0x49];
```

Comment:
```suggestion
    /* 0x144 */ char unk_144[0x4C];
```

Reply:
Thanks for catching this, forgot about the header 😅